docs/docsource/operation/Introduction/supported_clouds+services.rst

```diff
@@ -0,0 +1,76 @@
+Supported Clouds and Services
+=============================
+
+Zenko replicates and manages data from one or many sources to one or several
+destinations. Sources can be clouds or services, and targets can be of several
+types of cloud backend.
+
+Supported Sources
+-----------------
+
+Zenko supports multiple-cloud storage operations natively using the Amazon
+Simple Storage Service (S3) protocol. It can import data from many S3-compatible
```

Comment:
I think maybe `import data` gives the wrong idea here.

Reply:
Would "ingest" be more palatable? 
Ultimate vagueness: "handle"?

Reply:
Yeah "handle" might be the best in this situation.